src/main/java/io/vertx/gsoc2023/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -1,27 +1,106 @@
 package io.vertx.gsoc2023.qotd;
-
 import io.vertx.config.ConfigRetriever;
 import io.vertx.core.AbstractVerticle;
 import io.vertx.core.Promise;
+import io.vertx.core.eventbus.EventBus;
+import io.vertx.core.eventbus.MessageConsumer;
 import io.vertx.core.http.HttpServer;
 import io.vertx.core.http.HttpServerOptions;
+import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.BodyHandler;
 import io.vertx.pgclient.PgConnectOptions;
 import io.vertx.pgclient.PgPool;
 import io.vertx.sqlclient.PoolOptions;
+import io.vertx.sqlclient.Tuple;
+
 
 public class QuoteOfTheDayVerticle extends AbstractVerticle {
 
   private PgPool pgPool;
 
+
+
   @Override
   public void start(Promise<Void> startFuture) throws Exception {
     ConfigRetriever retriever = ConfigRetriever.create(vertx);
     retriever.getConfig().compose(config -> {
       pgPool = setupPool(config);
       var router = setupRouter();
+      EventBus eb = vertx.eventBus();
+      MessageConsumer<JsonArray> consumer = eb.consumer("realtime-feedback");
+
+      router.get("/quotes").handler((ctx) -> {
+        JsonArray quoteEntry = new JsonArray();
+        pgPool.query("SELECT  text FROM quotes").execute(
+          ar -> {
+            if (ar.succeeded())
+              ar.result().forEach(quote -> {
+                quoteEntry.add(
+                  new JsonObject()
+                    .put("quote", quote.getValue("text")));
+              });
+             else
+              ctx.response().setStatusCode(500)
```

Comment:
You need to stop the execution flow after sending the error reply. Otherwise, the server will try to send another response after.

Reply:
done